### PR TITLE
Removed JsonMessageSerializer and replaced by DataContractJsonSerializer

### PR DESCRIPTION
**Pull request by danielmarbach**
Since we plan to internalize the serializers the custom checks now use the built in DataContractJsonSerializer

@Particular/servicecontrol-maintainers please review

cc @andreasohlund @mat-mcloughlin

### CODE REVIEW

File: src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    
+	using NServiceBus;
+	
 
-    /*
+	/*
 		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
 		can be found here: http://particular.net/articles/the-nservicebus-host
 	*/
-    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```
Comment (andreasohlund) [inline src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs:11]: So we still need the sample in here? (or could we just have an acceptance test? => can be different pull if yes)

Comment (danielmarbach) [inline src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs:11]: I raised that question before https://github.com/Particular/ServiceControl.Plugin.Nsb6.CustomChecks/pull/2#issuecomment-176049412

Comment (andreasohlund) [inline src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs:11]: I still see value in acceptance tests, we can just fake SC and check that a message arrives and has the expected values. That would IMO catch most issues?

Comment (andreasohlund) [inline src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs:11]: I can take a stab at removing the sample and add a few acceptance tests instead?

**Pull request closed without merging**